### PR TITLE
Windows command line does not support single quotes.

### PR DESCRIPTION
**Pull request by BruceLee569**
System version: Windows 10 21H2

Python version: Python 3.9.5 (tags/v3.9.5:0a7dcbd, May  3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] on win32

At the command-line terminal (cmd or Powershell) you have to use double quotes to enclose the argument you wish to send. Enclose it with double quotes, and use single quotes within it. (https://stackoverflow.com/a/34379661/11687405)

For details, please refer to the GIF：https://gifyu.com/image/SLtLK

### CODE REVIEW

**Comment by fatihkabakk** [comment]
> System version: Windows 10 21H2
> 
> Python version: Python 3.9.5 (tags/v3.9.5:0a7dcbd, May 3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] on win32
> 
> At the command-line terminal (cmd or Powershell) you have to use double quotes to enclose the argument you wish to send. Enclose it with double quotes, and use single quotes within it. (https://stackoverflow.com/a/34379661/11687405)
> 
> For details, please refer to the GIF：https://gifyu.com/image/SLtLK

Yes, you're correct. However, powershell does not require double quotes. Powershell is like a simulation of `cmd` and `sh` together. To explain in detail, `cmd` requires every space-including parameter to be encolsed in double quotes. Also, see [The Developer Guide](https://devguide.python.org/).

**Comment by JelleZijlstra** [comment]
On Unix, single quotes are better because double quotes may do variable substitution. I'm not familiar enough with Windows to opine on what quotes are better there.

**Comment by zooba** [comment]
Maybe just say "single or double quotes"? This is a hint for people who are both new to terminals and new to Python, but it isn't really our responsibility to go too deep into how to use every single terminal in the world.

In general, Windows and cmd don't care about quotes, but the apps themselves do the splitting and might care. PowerShell expands variables within double quotes, but not within single quotes, but it's hard to say which may be preferred in any given case.

So basically, "it's complicated," and we should probably just avoid having too strong an opinion.

File: Doc/tutorial/interpreter.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ A second way of starting the interpreter is ``python -c command [arg] ...``,
 which executes the statement(s) in *command*, analogous to the shell's
 :option:`-c` option.  Since Python statements often contain spaces or other
 characters that are special to the shell, it is usually advised to quote
-*command* in its entirety with single quotes.
+*command* in its entirety with double quotes.
```
Comment (JelleZijlstra) [inline Doc/tutorial/interpreter.rst]:
```suggestion
*command* in its entirety.
```
Let's just avoid giving a precise recommendation (following @zooba's comment).

**Comment by JelleZijlstra** [comment]
Opened #98204 because of the CLA issues on this one. (The change we ended up with is distinct enough from @BruceLee569's original change that I don't think there's an issue with the new PR.)
